src/components/SamsungContainer.js

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchSamsungData } from '../redux/samsung/samsungSlice';
+import DataHandler from './DataHandler';
+import Header from './Header';
+
+const Samsung = () => {
```

Comment:
- [ ] It would be better if we create a `back` button that takes us to the home page when clicked. Although, we have a home button in the header performing the same functionality, however, we are required to follow the user stories to design the app
![Screenshot from 2023-06-27 23-11-30](https://github.com/ClintonYounge/Mobile-Tech-Financial-Digest/assets/46388113/ef5bd7a9-c0d3-4dcc-9c3b-2291365bab3a)